addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -440,22 +441,20 @@ def get_empty_list_help(self, help_message):
         else:
             alias_id = False
 
-        nocontent_values = {
+        nocontent_body = Markup(
+            """
+<p class="o_view_nocontent_empty_folder">%(help_title)s</p>
+<p>%(para_1)s<br/>%(para_2)s</p>""" % {
```

Comment:
```suggestion
<p>%(para_1)s<br/>%(para_2)s</p>""") % {
```
you can move the labels outside of markup if it does not contain html code

Reply:
Done Sir.